chatminer/chatparsers.py

```diff
@@ -68,15 +68,15 @@ def __init__(self, filepath):
     def parse_file(self):
         self._logger.info("Starting reading raw messages...")
         self._read_raw_messages_from_file()
-        self._logger.info("Finished reading %i raw messages.", len(self._raw_messages))
+        self._logger.info(f"Finished reading {len(self._raw_messages)} raw messages.")
```

Comment:
I used f-strings before, but was made aware that the less beautiful %-strings have better performance in logging scenarios due to lazy loading. See [here](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation) or [here](https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs) for reference.

Reply:
I will fix this in the next commit.